packages/cli/src/cmds/resource.ts

```diff
@@ -339,11 +340,11 @@ export class ResourceList extends YargsCommand {
     pluginNameMap.set("fx-resource-function", "azure-function");
 
     if (result.isOk()) {
-      pluginNameMap.forEach((pluginAlias: string, pluginName: string) => {
+      for (const [pluginAlias, pluginName] of pluginNameMap) {
         if (pluginName in result.value) {
-          console.log(pluginAlias);
+          await CLILogProvider.necessaryLog(LogLevel.Info, pluginAlias, true);
```

Comment:
It seems the behavior is changed here. I don't know whether this is what you expect.

Originally this is `console.log()`, but the new one is `console.log(s.white())`.
When the user's default color is not white, these strings will be white.

Reply:
Yeah, the colors of logs will be defined more clearly after. Now the white logs should be the main color theme in the CLI.